crates/swc_ecma_minifier/tests/fixture/issues/quagga2/1.4.2/1/output.js

```diff
@@ -2632,7 +2632,7 @@
                 "use strict";
                 var undefined, Op = Object.prototype, hasOwn = Op.hasOwnProperty, $Symbol = "function" == typeof Symbol ? Symbol : {}, iteratorSymbol = $Symbol.iterator || "@@iterator", asyncIteratorSymbol = $Symbol.asyncIterator || "@@asyncIterator", toStringTagSymbol = $Symbol.toStringTag || "@@toStringTag";
                 function wrap(innerFn, outerFn, self, tryLocsList) {
-                    var protoGenerator = outerFn && outerFn.prototype instanceof Generator ? outerFn : Generator, generator = Object.create(protoGenerator.prototype), context = new Context(tryLocsList || []);
+                    var generator = Object.create((outerFn && outerFn.prototype instanceof Generator ? outerFn : Generator).prototype), context = new Context(tryLocsList || []);
```

Comment:
Original source code:

```js
                        var protoGenerator =
                            outerFn && outerFn.prototype instanceof Generator
                                ? outerFn
                                : Generator;
                        var generator = Object.create(protoGenerator.prototype);
                        var context = new Context(tryLocsList || []); // The ._invoke method unifies the implementations of the .next,
                        
```

Reply:
So this is fine. `Object.create` does not modify `protoGenerator` or the initializer (`outerFn && outerFn.prototype instanceof Generator
                                ? outerFn
                                : Generator`)